lib/rubocop/cop/lint/non_deterministic_require_order.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Dir[...] and Dir.glob(...) do not make any guarantees about
```

Comment:
Please put the `Dir...` fragments in single backticks in order to format them as code.